lms/views/onedrive.py

```diff
@@ -35,4 +35,8 @@ def verify_domain(request):
       ]
     }}"""
     )
-    return Response(text=response_text, content_type="application/json")
+    return Response(
+        text=response_text,
+        content_type="application/json",
+        headers={"Content-Length": str(len(response_text))},
```

Comment:
You can lie here (ie set it to some random value) and that's what you get, so at least this take preference over any other mechanism at play.